documentation/asciidoc/accessories/camera/synchronous_cameras.adoc

```diff
@@ -1,102 +1,97 @@
 
-Solder 2 wires to the XVS test points on each board and connect both of these wires together to the 1.65V potential divider.
+=== Libcamera configuration
 
-Solder the GND of each Camera board to each other. Also solder 2 wires to the XHS test points on each board and connect these. No pullup is needed for XHS pin.
+If the cameras are not all started within 1 second, the `rpicam-vid` application can time out. To prevent this, you must edit a configuration file on any Raspberry Pi(s) with sink cameras. On Raspberry Pi 5 or CM5, make a copy of the file `/usr/share/libcamera/pipeline/rpi/pisp/example.yaml`. On other Raspberry Pi models, make a copy of the file `/usr/share/libcamera/pipeline/rpi/vc4/rpi_apps.yaml`.
 
-On the boards that you wish to act as sinks, solder the two halves of the MAS pad together. This tells the sensor to act as a sink, and will wait for a signal to capture a frame.
+In both cases, delete the `#` (comment) from the `"camera_timeout_value_ms":` line, and change the number to `60000` (60 seconds).
 
-==== Boot up source and sink
+=== Starting the cameras
 
-Run the following command to start the sink:
+Run the following commands to start the sink:
 
-[source,console]
+[,bash,subs=+quotes]
 ----
-$ rpicam-vid --frames 300 -o sync.h264
+export LIBCAMERA_RPI_CONFIG_FILE=_edited_copy_of_rpi_apps.yaml_
+rpicam-vid --frames 300 --qt-preview -o sink.h264
 ----
 
-Due to the limitations of the IMX296 sensor, the sink cannot record exactly the same number of frames as the source. **The source records one extra frame before the sink starts recording**. Because of this, you need to specify that the sink records one less frame with the `--frames` option.
-
-Wait at least two seconds before you start the source.
-
-After waiting two seconds, run the following command to start the source:
+Wait a few seconds, then run the following command to start the source:
 
-[source,console]
+[,bash,subs=+quotes]
 ----
-$ rpicam-vid --frames 299 -o sync.h264
+rpicam-vid --frames 300 --qt-preview -o source.h264
 ----
+Frames should be synchronised. Use `--frames` to ensure the same number of frames are captured, and that the recordings are exactly the same length.
+Running the sink first ensures that no frames are missed.
 
-Because the sink and source record a different number of frames, use `ffmpeg` to resync the videos. By dropping the first frame from the source, we then get two recordings with the same starting point and frame length:
-
-[source,console]
-----
-$ ffmpeg -i source.h264 -vf select="gte(n\, 1)" source.h264
-----
+NOTE: When using the GS camera in synchronous mode, the sink will not record exactly the same number of frames as the source. **The source records one extra frame before the sink starts recording**. Because of this, you need to specify that the sink records one less frame with the `--frames` option. You could use `ffmpeg` to delete the first frame from the source camera's video recording.
```

Comment:
"You could use" -> "Alternatively you could use"

Reply:
Not only but also!

I need to test this again because getting it wrong leads to a (soft) lockup if the sink doesn't get the right number of syncs.

Reply:
> Not alternatively, but as well as!

Ahhh, in which case perhaps the "could" ought to be "should then" ? (and perhaps worth bringing back the example ffmpeg command line)

Reply:
Yes. I was trying to corral the GS-Cam-specific bits into NOTEs, but the ffmpeg command can't really go inside one of those, so might have to re-jig the structure.

It would be nice if we could obliterate GS Camera from this part, but somebody is bound to be using it already...

Reply:
Well it would be nice to steer GS Cam users towards the alternative "external trigger" method (which requires fewer wires, is software-configurable, and doesn't have the asymmetry / off-by-one-frame issues)

Sadly, now that we've documented this method we can't really un-document it.

Reply:
Ahhh, now I see what you're saying :slightly_smiling_face:  Yeah, I guess all we can do is strongly-discourage it.

Reply:
I've tentatively removed the `ffmpeg` bit...